motor_controllers/kostal_inveor/manifest.yml

```diff
@@ -0,0 +1,114 @@
+blueprint_spec: device/1.0
+
+display_name: KOSTAL INVEOR
+description: Variable frequency drive and motor controller
+icon: enapter-motor-controller
+vendor: kostal
+author: enapter
+contributors:
+  - anataty
+  - alexandershalin
+support:
+  url: https://go.enapter.com/enapter-blueprint-support
+  email: support@enapter.com
+license: MIT
+verification_level: ready_for_testing
+
+communication_module:
+  product: ENP-RS485
+  lua:
+    file: firmware.lua
+    dependencies:
+      - enapter-ucm
+
+properties:
+  vendor:
+    type: string
+    display_name: Vendor
+  model:
+    type: string
+    display_name: Model
+
+telemetry:
+  status:
+    display_name: Status
+    type: string
+    enum:
+      - ok
+      - read_error
+  target_freq:
+    type: float
+    unit: Hz
+    display_name: Target Frequency
+  actual_freq:
+    type: float
+    unit: Hz
+    display_name: Actual Frequency
+  output_volt:
+    type: float
+    unit: V
+    display_name: Output Voltage
+  motor_curr:
+    type: float
+    unit: A
+    display_name: Motor Current
+  igbt_temp:
+    type: float
+    unit: °C
+    display_name: IGBT Temperature
+  inner_temp:
+    type: float
+    unit: °C
+    display_name: Inner Temperature
+
+command_groups:
+  config:
+    display_name: Configuration
+commands:
+  write_configuration:
+    populate_values_command: read_configuration
+    display_name: Configure
+    group: config
+    ui:
+      icon: wrench-outline
+    arguments:
+      address:
+        display_name: Modbus address
+        description: 1-247
```

Comment:
you can set [min and max](https://developers.enapter.com/docs/reference#command-arguments) for this argument